os/mod.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Returns the current user's home directory.
+ * On Unix, including macOS, it returns the $HOME environment variable.
+ * On Windows, it returns %USERPROFILE%.
+ *
+ * Ported from Go: https://github.com/golang/go/blob/go1.12.5/src/os/file.go#L389
```

Comment:
Add "Needs permissions to access env (--allow-env)."

Reply:
Done.